doc/source/serve/direct-ingress.md

```diff
@@ -0,0 +1,102 @@
+# Direct Ingress
+
+In the 2.1, Serve provides an alpha version of gRPC ingress. In this section, you will learn how to
+
+* use Serve's internal gRPC schema to send traffic
+* bring your own gRPC schema into your Serve application
+
+## Use Serve's Schema
+
+Internally, Serve provides a simple gRPC schema.
```

Comment:
```suggestion
Serve provides a simple gRPC schema to machine learning inference workload. It is designed to be kept simple and you are encouraged to adapt it for your own need. 
```

Reply:
It is designed to be kept simple, and you are encouraged to adapt it to your own need